network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/VpnRemoteFeatures.kt

```diff
@@ -40,6 +40,7 @@ import kotlinx.coroutines.launch
     scope = AppScope::class,
     featureName = "networkProtection",
     toggleStore = VpnRemoteFeaturesStore::class,
+    settingsStore = VpnRemoteSettingsStore::class,
```

Comment:
Should we use the newly added `config` block in sub-features?

Reply:
Discussed this in zoom and decided that it doesn’t make sense to use `config`.